src/Frontend/Core/Engine/Footer.php

```diff
@@ -43,26 +43,26 @@ public function parse(): void
         $footerLinks = (array) Navigation::getFooterLinks();
         $this->template->assignGlobal('footerLinks', $footerLinks);
 
-        $siteHTMLFooter = (string) $this->get('fork.settings')->get('Core', 'site_html_footer', null);
+        $siteHTMLEndOfBody = (string) $this->get('fork.settings')->get('Core', 'site_html_end_of_body', $this->get('fork.settings')->get('Core', 'site_html_footer', null));
         $facebookAppId = $this->get('fork.settings')->get('Core', 'facebook_app_id', null);
 
         // facebook admins given?
         if ($facebookAppId !== null) {
             // add Facebook container
-            $siteHTMLFooter .= $this->getFacebookHtml($facebookAppId);
+            $siteHTMLEndOfBody .= $this->getFacebookHtml($facebookAppId);
         }
 
         // add Google sitelinks search box code if wanted.
         if ($this->get('fork.settings')->get('Search', 'use_sitelinks_search_box', true)) {
             $searchUrl = FrontendNavigation::getUrlForBlock('Search');
             $url404 = FrontendNavigation::getUrl(Model::ERROR_PAGE_ID);
             if ($searchUrl !== $url404) {
-                $siteHTMLFooter .= $this->getSiteLinksCode($searchUrl);
+                $siteHTMLEndOfBody .= $this->getSiteLinksCode($searchUrl);
             }
         }
 
         // assign site wide html
-        $this->template->assignGlobal('siteHTMLFooter', $siteHTMLFooter);
```

Comment:
for now I would assign it to the both of them for BC reasons. There are people that upgrade to the latest version of fork and this will break the theme that they use

Reply:
Fixed in dabf936ae918db1eac41e60b3cfe98587fc8d0aa